### PR TITLE
use UNSPECIFIED for socketAddr and reduce a parameter for the Builder::new

### PR DESCRIPTION
**Pull request by zhlsunshine**
1. Use UNSPECIFIED for socketAddr of Default config 
2. Reduce a parameter for the Builder::new

### CODE REVIEW

File: src/admin/mod.rs [hunk 1]
```diff
@@ -44,9 +44,9 @@ pub struct Server {
 pub struct Readiness(Arc<AtomicBool>);
 
 impl Builder {
-    pub fn new(addr: SocketAddr, f: WorkloadInformation) -> Self {
+    pub fn new(f: WorkloadInformation) -> Self {
```
Comment (hzxuzhonghu) [inline src/admin/mod.rs:47]: Maybe need to pass address later

**Review by hzxuzhonghu — COMMENTED** [review]
LGTM othewise

File: src/admin/mod.rs (hunk 1 above)
Comment (howardjohn) [reply src/admin/mod.rs:47]: Yeah we should put this in Config. Even though it's not configurable right now, in the future we will make everything in Config customizable

Comment (zhlsunshine) [reply src/admin/mod.rs:47]: Okay, got it. I handled the conflict and re-submitted.

**Review by howardjohn — APPROVED** [review]

**Pull request merged**